taxi/main.go

```diff
@@ -0,0 +1,12 @@
+package main
+
+import (
+	"fmt"
+	"taxi/system"
+)
+
+func main() {
+	logs := "13:50:08.245 0.0<LF>13:50:11.123 4.0<LF>13:50:12.125 10.2<LF>13:50:13.100 8.7<LF>"
+	totalFare, _ := system.CalcTotalFare(logs)
+	fmt.Println(totalFare)
```

Comment:
You should handle error cases correctly.
Check this:
https://github.com/line/recruiting-resources-jp/tree/main/techtrain/1st-mission#%E5%87%BA%E5%8A%9B%E5%BD%A2%E5%BC%8F